public/markdown/contributing.md

```diff
@@ -41,10 +40,10 @@ Before making a contribution, it is important to make sure that the change you w
         * Features are forever, always think about whether a new feature really belongs to the core framework or if it should be implemented as a module
         * Code must conform to standard style guidelines and pass all tests (see [Run tests](documentation/latest/BuildingFromSource#run-tests))
     6. New files must:
-        * Have a Lightbend copyright header in the style of ``Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>``.
+        * Have a copyright header in the style of ``2011-2021 Lightbend Inc., since 2022 the original author or authors from the Play Framework project``.
```

Comment:
I don't get that this sentence means. 

I also don't know what we should do with the copyright header. I guess all existing files should keep the Lightbend copyright. After they are edited, they should include Lightbend and something else, eg: Play Framework Organisation. 

New files can have **Play Framework Organisation** only. 

We need to learn more about it.

Reply:
The sentence is similar to what JHipster uses, please see our conversation on slack we had 2 month ago: https://typesafe.slack.com/archives/CFPVDLR41/p1637077983073200

Reply:
I thought about changing the copyright of all files on the play `master` branch to this wording (or similiar). So the 2.8.x branch still is "under control" by Lightbend, however upcoming 2.9 is community. Also the sentences says "unitl 2021 this is Lightbend stuff, after that community"

Reply:
Anyway, I will revert this sentence for now but deploy the rest already.

Reply:
Renato look here:
https://github.com/jhipster/generator-jhipster/blob/main/cli/commands.js#L2-L5

Reply:
Lets track that here: https://github.com/playframework/play-meta/issues/211